src/tokens/selectors.ts

```diff
@@ -356,8 +355,7 @@ export const swappableTokensByNetworkIdSelector = createSelector(
 
 export const cashInTokensByNetworkIdSelector = createSelector(
   (state: RootState, networkIds: NetworkId[]) => tokensListSelector(state, networkIds),
-  (tokens) =>
-    tokens.filter((tokenInfo) => tokenInfo.isCashInEligible && isCicoToken(tokenInfo.symbol))
```

Comment:
> Maybe the safest thing to do would be to retain this check if the bottom sheet feature gate is true

wouldn't the check be needed when the bottom sheet feature gate is false? since that's when the old screen is shown which doesn't handle tokens where `isCicoToken` evaluates to false?

Reply:
@satish-ravi

Reply:
oops yes, I meant when the feature gate is false